backend/.env.staging.example

```
@@ -0,0 +1,6 @@
+# MCP — Staging environment
+# Copy to .env.staging and fill real credentials (never commit .env.staging)
+DATABASE_URL="postgresql://tasktime:CHANGE_ME@5.129.242.171:5432/tasktime?schema=public"
+REDIS_URL="redis://5.129.242.171:6379"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)